blocks/bricks/conv.py

```diff
@@ -52,13 +52,13 @@ def __init__(self, filter_size, num_filters, num_channels, batch_size=None,
         self.border_mode = border_mode
 
     def _allocate(self):
-        W = shared_floatx_zeros((self.num_filters, self.num_channels) +
+        W = shared_floatx_nans((self.num_filters, self.num_channels) +
                                 self.filter_size, name='W')
```

Comment:
You'll need to remove one space in this line, and in many of the others, so that the indention matches again.